WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -8,6 +8,10 @@ struct DefaultProductFormTableViewModel: ProductFormTableViewModel {
     private let currency: String
     private let currencyFormatter: CurrencyFormatter
 
+    // Timezone of the website
+    //
+    var siteTimezone: TimeZone = TimeZone.websiteTimezone
```

Comment:
nit: can this be changed to ` = Timezone.siteTimezone`? It would be consistent with the naming used everywhere else.